### ISSUE TITLE
When running ComfyUI, two types of warnings will appear:

### ISSUE THREAD

**Issue by ye-pei-sheng** [city96/ComfyUI-GGUF#66]
When running ComfyUI, two types of warnings will appear:
1. ComfyUI-GGUF\nodes.py:35: UserWarning: The given NumPy array is not writable, and PyTorch does not support non-writable tensors. This means writing to this tensor will result in undefined behavior. You may want to copy the array to protect its data or make it writable before converting it to a tensor. This type of warning will be suppressed for the rest of this program. (Triggered internally at C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\torch\csrc\utils\tensor_numpy.cpp:212.)
  torch.from_numpy(tensor.data), # mmap

2. ComfyUI-GGUF\dequant.py:8: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).

=================
After searching online, it seems that the second warning can be eliminated by modifying dequant.py:8 to change 
data = torch.tensor(tensor.data)  to 
data = torch.as_tensor(tensor.data)

**Comment by city96** [comment]
Doing that will preserve the extra metadata on the tensor and (possibly) make dequantization slower. The torch.tensor() thing is done to avoid that by converting it back to a normal tensor.

![image](https://github.com/user-attachments/assets/03472838-5741-4958-9ee4-0d42e0327ba2)

The first warning is due to mmap, and it was done like that in order to avoid taking 2x system memory on load by keeping it write-only (we can't write to quantized weights anyway) though looks like that logic got broken by ComfyUI at some point so it still takes 2x memory anyway.

Neither of the warnings should affect performance and should only show up once.

**Comment by ye-pei-sheng** [comment]
@city96 Thank you for your reply. I have learned a lot again

**Comment by city96** [comment]
No worries! I'll leave this issue open and hopefully find a better solution for both of those things in the future.
